app/javascript/redux/reducer.js

```diff
@@ -0,0 +1,26 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import axios from "axios";
+
+const fetchGeetingAsyncThank = createAsyncThunk("greetings/fetchGeeting", async () => {
+  const response = await axios.get('http://127.0.0.1:5000/api/v1/greetings')
+                              .catch(() => false);
+  return response.data;
+});
+
+const initialState = {
+  greetings: []
+};
+
+const greetReducer = createSlice({ 
+  name: "greeting",
+  initialState,
+  extraReducers: (builder) => {
+    builder.addCase(fetchGeetingAsyncThank.fulfilled, (state, {payload}) => {
+      state.greetings = payload;
+    });
+  }
+});
+
+const getAllGreetings = (state) => state.greeting.greetings;
```

Comment:
- [ ] Great job so far but there is some error that you need to fix.
- After run to app I'm seeing this 👇🏻 

![Screenshot from 2023-01-28 18-55-38](https://user-images.githubusercontent.com/79058364/215279108-59c84a19-b955-4616-b5d4-bd17cc64c98a.png)

- Based on the project requirement, the  React app should use the URL and displayes a random greeting however I'm getting this error instead when I reload the page  👇🏻  please fix this 
![Screenshot from 2023-01-28 18-55-59](https://user-images.githubusercontent.com/79058364/215279113-45fee769-b5cf-44e1-aa51-48d27d1972b0.png)